src/commands/fts/SearchWorkbench/searchWorkbench.ts

```diff
@@ -47,29 +74,52 @@ export class SearchWorkbench {
                 await workbenchWebviewProvider.sendQueryResult(JSON.stringify([{ "status": "Executing statement" }]), { queryStatus: QueryStatus.Running }, null);
                 const explainPlan = JSON.stringify("");
                 const couchbbaseRestAPI = new CouchbaseRestAPI(connection);
+                const searchIndexesManager = connection?.cluster?.searchIndexes();
```

Comment:
Why no scope level indexes?

Reply:
This is better in this file, i feel because: queryContext?.indexName is of the format bucketName.scopeName.indexName and using scope level indexes returns just indexName. Also we will need to pass additional parameter scopeName and bucketName to this method which is unnecessary, as we can just check directly if the index is present in the cluster level or not.